hazelcast/src/test/java/com/hazelcast/jet/impl/execution/init/JetJobPrunabilityTest.java

```diff
@@ -140,8 +140,14 @@ public void test_simpleDag_takesMemberAndCoordinator() {
 
         assertJobStatusEventually(job, JobStatus.COMPLETED);
         List<List<Object>> lists = consumerPms.getLists();
+        List<List<Object>> nonEmptyRes = lists.stream().filter(l -> !l.isEmpty()).collect(Collectors.toList());
+
         // one of the processors should get all data
-        assertThat(lists).contains(List.of(0, 1, 0, 1));
+        assertThat(nonEmptyRes).isNotEmpty();
```

Comment:
@Fly-Style can similar problem happen in test_simpleDag_onlyCoordinator?

Reply:
yes, it can. Will fix it.